### PR TITLE
max recipients is now optionally set-able by caller; 7 by default

### PR DESCRIPTION
**Pull request by elsehow**
not sure why 255 is the magic number for maximum possible recipients. the number is quite reminiscent of 256 / 8*32 ....

### CODE REVIEW

**Comment by dominictarr** [comment]
can you make maxrecipients camel case? or, you can make that a one liner with `max = max || 7`

The maximum value for `max` is 255 because it's stored in a single byte. A way you could extend this, is you could estimate the maximum recipient size from the size of the message, since messages over 8k _could_ contain more than 255 recipients.

However... that was never really the intended use of private-box, and the code is very very simple, so I'd wouldn't recommend using private box for messages with a large number of recipients.

**Comment by elsehow** [comment]
we can throw an error when someone tries to specify more than 255 recipients for now

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "private-box",
   "description": "encrypt a message to a secret number of recipients",
-  "version": "0.1.1",
+  "version": "0.1.2",
```
Comment (dominictarr) [inline package.json]: you shouldn't set the version number when making a PR. that is something the maintainer sets after merging. they may merge multiple PRs and then do one publish, so better to leave that to them.

**Comment by elsehow** [comment]
got it - fixed

**Comment by dominictarr** [comment]
thanks, merged into 0.2.0!

**Pull request merged**